WebDriverAgentLib/Utilities/FBConfiguration.m

```diff
@@ -44,10 +44,22 @@
 static BOOL FBIncludeNonModalElements = NO;
 static NSString *FBAcceptAlertButtonSelector = @"";
 static NSString *FBDismissAlertButtonSelector = @"";
+static NSString *FBSnapshotMaxDepthKey = @"maxDepth";
+static NSMutableDictionary *FBSnapshotRequestParameters;
 
 
 @implementation FBConfiguration
 
++ (void)initialize
+{
+  FBSnapshotRequestParameters = [NSMutableDictionary dictionaryWithDictionary:@{
+    @"maxArrayCount": @INT_MAX,
+    @"maxChildren": @INT_MAX,
+    @"maxDepth": @50, // 50 should be enough for majority cases. Up to 100 performance is acceptable.
```

Comment:
for the majority of the cases

Reply:
The performance is acceptable for values up to `100`

Reply:
done!